editors/moe/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                moe
+version             1.12
+categories          editors
+platforms           darwin
+license             GPL-3
+maintainers         {aeiouaeiouaeiouaeiouaeiouaeiou @aeiouaeiouaeiouaeiouaeiouaeiou} openmaintainer
```

Comment:
You don't have a macports.org email address so you can't use shorthand here and need to list your email address as described in the MacPorts documentation.